src/crackmes/ioli/ioli_0x01.md

```diff
@@ -27,44 +30,39 @@ $ r2 ./crackme0x01
 -- Use `zoom.byte=printable` in zoom mode ('z' in Visual mode) to find strings
 [0x08048330]> aa
 [0x08048330]> pdf@main
-/ (fcn) main 113
-|          ; var int local_4 @ ebp-0x4
-|          ; DATA XREF from 0x08048347 (entry0)
-|          0x080483e4    55           push ebp
-|          0x080483e5    89e5         mov ebp, esp
-|          0x080483e7    83ec18       sub esp, 0x18
-|          0x080483ea    83e4f0       and esp, -0x10
-|          0x080483ed    b800000000   mov eax, 0
-|          0x080483f2    83c00f       add eax, 0xf
-|          0x080483f5    83c00f       add eax, 0xf
-|          0x080483f8    c1e804       shr eax, 4
-|          0x080483fb    c1e004       shl eax, 4
-|          0x080483fe    29c4         sub esp, eax
-|          0x08048400    c7042428850. mov dword [esp], str.IOLI_Crackme_Level_0x01_n ; [0x8048528:4]=0x494c4f49  ; "IOLI Crackme Level 0x01." @ 0x8048528
-|          0x08048407    e810ffffff   call sym.imp.printf
-|             sym.imp.printf(unk)
-|          0x0804840c    c7042441850. mov dword [esp], str.Password_ ; [0x8048541:4]=0x73736150  ; "Password: " @ 0x8048541
-|          0x08048413    e804ffffff   call sym.imp.printf
-|             sym.imp.printf()
-|          0x08048418    8d45fc       lea eax, dword [ebp + 0xfffffffc]
-|          0x0804841b    89442404     mov dword [esp + 4], eax ; [0x4:4]=0x10101
-|          0x0804841f    c704244c850. mov dword [esp], 0x804854c ; [0x804854c:4]=0x49006425  ; "%d" @ 0x804854c
-|          0x08048426    e8e1feffff   call sym.imp.scanf
-|             sym.imp.scanf()
-|          0x0804842b    817dfc9a140. cmp dword [ebp + 0xfffffffc], 0x149a
-|      ,=< 0x08048432    740e         je 0x8048442
-|      |   0x08048434    c704244f850. mov dword [esp], str.Invalid_Password__n ; [0x804854f:4]=0x61766e49  ; "Invalid Password!." @ 0x804854f
-|      |   0x0804843b    e8dcfeffff   call sym.imp.printf
-|      |      sym.imp.printf()
-|     ,==< 0x08048440    eb0c         jmp 0x804844e ; (main)
-|     ||   ; JMP XREF from 0x08048432 (main)
-|     |`-> 0x08048442    c7042462850. mov dword [esp], str.Password_OK____n ; [0x8048562:4]=0x73736150  ; "Password OK :)." @ 0x8048562
-|     |    0x08048449    e8cefeffff   call sym.imp.printf
-|     |       sym.imp.printf()
-|     |    ; JMP XREF from 0x08048440 (main)
-|     `--> 0x0804844e    b800000000   mov eax, 0
-|          0x08048453    c9           leave
-\          0x08048454    c3           ret
+            ; DATA XREF from entry0 @ 0x8048347
+┌ 113: int main (int argc, char **argv, char **envp);
```

Comment:
can you disable `utf-8` and update again this section?

Reply:
sure, I disabled it and updated again ~